File: PRAK404-2210817120011-MAULIDASARI.c
#include <stdio.h>
int main()
{
    int nilai;
	double a, b; 
	char pilihan;

	do{
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        
		printf("Masukkan Pilihan : ");
		scanf("%d", &nilai);
        // 
        if (nilai > 5 ) {
        
        } else if (nilai == 5){
           printf("Terimakasih, telah menggunakan kalkulator MAULIDASARI ");
             break;
        } else {
        printf("Masukkan Nilai Pertama : ");
        scanf("%lf", &a);
        printf("Masukkan Nilai Kedua : ");
        scanf("%lf", &b);
        }
        
		switch (nilai)
		{
		case 1:
			printf("Hasil Penjumlahan antara %.2lf dengan %.2lf adalah %.2lf",a,b, a + b);
			break;
		case 2:
			printf("Hasil Pengurangan antara %.2lf dengan %.2lf adalah %.2lf",a,b, a - b);
			break;
		case 3:
			printf("Hasil Perkalian antara %.2lf dengan %.2lf adalah %.2lf",a,b, a * b);
			break;
		case 4:
			if (b == 0)
				printf("\n nilai kedua tidak boleh 0");
			else
				printf("Hasil pembagian antara  %.2lf dengan %.2lf adalah %.2lf",a,b, a / b); // b tidak boleh kosong
			break;
		default:
			printf("Input anda salah, silahkan coba lagi");
            break;
		}
		printf("\n Ulang lagi? (y/t)");
		scanf(" %c", &pilihan);
	} while (pilihan == 'y');
	return 0;
}

File: PRAK404-2210817120011-MAULIDASARI.py
while True:  
    choice = input("Pilih Program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\nMasukkan Pilihan :")
    
    if choice == '1':
        n1 = float(input("Masukkan nilai pertama : "))
        n2 = float(input("Masukkan nilai kedua : "))
        sum = n1 + n2
        print("Hasil penjumlahan adalah ",sum)

    elif choice == '2':
        n1 = float(input("Masukkan nilai pertama : "))
        n2 = float(input("Masukkan nilai kedua : "))
        diff = n1 - n2
        print("Hasil pengurangan adalah ",diff)
    
    elif choice == '3':
        n1 = float(input("Masukkan nilai pertama : "))
        n2 = float(input("Masukkan nilai kedua : "))
        prod = n1 * n2
        print("Hasil perkalian adalah ",prod)
    elif choice == '4':
        n1 = float(input("Masukkan nilai pertama : "))
        n2 = float(input("Masukkan nilai kedua : "))
        div = n1 / n2
        print("Hasil pembagian adalah",div)
    
    elif choice == '5':
        print("Terimakasih, telah menggunakan kalkulator MAULIDASARI ")
        break
    
    else:
        print("Input anda salah, silahkan coba lagi ")